metadata-service/factories/src/test/java/com/linkedin/metadata/boot/steps/IngestDataPlatformInstancesStepTest.java

```diff
@@ -46,7 +46,7 @@ public void testExecuteDoesNothingWhenDataPlatformInstanceAspectsAlreadyExists()
 
     verify(migrationsDao, times(1)).checkIfAspectExists(anyString());
     verifyNoMoreInteractions(migrationsDao);
-    verifyZeroInteractions(entityService);
```

Comment:
Out of curiosity, why did this change?

Reply:
This was a deprecated method previously and it's a transitive dep that got upgraded.